metrics/prometheus.go

```diff
@@ -42,7 +42,7 @@ func (p *Prometheus) Configure() {
 			// Result: true if the run was successful, false otherwise
 			"success",
 			// True if full run, false if quick run
-			"full_run",
+			"run_type",
```

Comment:
Hmmm... I was just following the naming conventions for the metrics that come with prometheus by default. but those conventions seem more for the metrics themselves, I can't find a convention for labels [https://prometheus.io/docs/practices/naming/](https://prometheus.io/docs/practices/naming).

Also, doing some more research into prometheus conventions is making me reflect that we may not be using labels idiomatically, but I don't want to worry too much about that right now.